pyvrp/cpp/educate/LocalSearch.cpp

```diff
@@ -312,7 +327,7 @@ LocalSearch::LocalSearch(ProblemData &data,
       orderRoutes(data.numVehicles()),
       lastModified(data.numVehicles(), -1),
       clients(data.numClients() + 1),
-      routes(data.numVehicles(), data),
+      routes(),
```

Comment:
Why touch this?

Reply:
Since i added the route idx to the constructor for Route